llvm/lib/Target/AArch64/AArch64LoadStoreOptimizer.cpp

```diff
@@ -87,6 +87,10 @@ static cl::opt<unsigned> LdStConstLimit("aarch64-load-store-const-scan-limit",
 static cl::opt<bool> EnableRenaming("aarch64-load-store-renaming",
                                     cl::init(true), cl::Hidden);
 
+// Enable SVE fill/spill pairing for VLS 128.
+static cl::opt<bool> EnableSVEFillSpillPairing("aarch64-sve-fill-spill-pairing",
+                                               cl::init(true), cl::Hidden);
```

Comment:
Given the transformation is little endian specific and the option's primary use case being for debug, perhaps it's worth reversing the polarity and implementing `DisableSVEFillSpillPairing -> aarch64-disable-sve-fill-pairing`?

Reply:
Just going to through this out there but when inspecting `tryToPairLdStInst` I see there is already provision to disable the use of pair instructions via target features `disable-ldp` and `disable-stp` so perhaps we don't need a dedicated flag after all?  That might make the implementation easier because then the "should we do this" check only requires the subtarget?